packages/localization/src/dates/modifyDateBy.ts

```diff
@@ -39,4 +39,59 @@ const modifyDateBy = (date: CalendarDate, amount: number, unit: string, minDate?
 	return newDate;
 };
 
+/**
+ * Adds or subtracts a given amount of days/months/years from a date.
+ * If minDate or maxDate are given, the result will be enforced within these limits
+ *
+ * <b>Note:</b> Used in the case when user is navigating with the mouse through the arrows of the Calendar header.
```

Comment:
I did not check why this is needed, but the name of the method "modifyDateByClick" and the docs "Used in the case when user is navigating with the mouse through the arrows of the Calendar header" makes the method very specific and can't be part of the localization package at least with this name and description.

The other method **modifyDateBy** adds and subtracts  days/months/years and restricts the result in a range of min and max and does not care what is clicked how it's clicked just calculating dates

The new one **modifyDateByClick** does the same from what I see but a little bit different, the result is a little bit different. Instead of explaining that the method is used when clicking the CalendarHeader, explain what it does.

Because now we see two methods 
- that modifies dates
- that both has this doc:
Adds or subtracts a given amount of days/months/years from a date.
 * If minDate or maxDate are given, the result will be enforced within these limits
 - but with different result. 

**What should be explained is the algorithm that is applied in one and in the other.**
Then we can see if having a second method is justified, or we can just add a parameter to the existing method or add the logic in another layer. 

*P.S. Also, because it was a single method, we used the default export. If we decide to have two methods, then we probably will need to export both with named exports.

Reply:
Hi @ilhan007,

Considering your comment and taking into account your feedback, I think the suggestion you pointed out about adding a parameter to the existing method, is more cleaner and better solution.
Thank you.

Now the focus on the calendar works as it should.
Please could you take a look if the solution seems okay to you ?

BR,
Stoyan